requirements.txt

```diff
@@ -6,7 +6,7 @@ numpy
 psutil
 pyyaml
 requests
-setuptools
+setuptools==72.1.0
```

Comment:
Please add a comment why one needs to specify an explicit version here? And wouldn't `<=72.1.0` be better?

Reply:
Done